wiki/Community/Moscow_osu!_Event/en.md

```diff
@@ -24,8 +24,8 @@
 
 Until 2023, large-scale osu!-related events were rarely held in Russia. Usually, these were small community meetups of 10–20 people, but many were interested in more significant events.
 
-Inspired by [cavoe's osu! event](/wiki/Community/cavoe's_osu!_event), the `
-*#russian osu cup tournament* teams and the [play-osu.ru](https://play-osu.ru) communities decided in 2023 to organise their own event in Moscow, primarily dedicated to the [osu! game mode](/wiki/Game_mode/osu!). In subsequent iterations, other modes were included as well.
+Inspired by [cavoe's osu! event](/wiki/Community/cavoe's_osu!_event), the 
+`#russian` osu cup tournament teams and the [play-osu.ru](https://play-osu.ru) communities decided in 2023 to organise their own event in Moscow, primarily dedicated to the [osu! game mode](/wiki/Game_mode/osu!). In subsequent iterations, other modes were included as well.
```

Comment:
either italics or quotes work to signify that this is supposed to be interpreted as a name (though linking the thing would work better)

the new line was a mistake

```suggestion
Inspired by [cavoe's osu! event](/wiki/Community/cavoe's_osu!_event), the "#russian osu! cup" tournament teams and the [play-osu.ru](https://play-osu.ru) communities decided in 2023 to organise their own event in Moscow, primarily dedicated to the [osu! game mode](/wiki/Game_mode/osu!). In subsequent iterations, other modes were included as well.
```